charts/alfresco-sync-service/Chart.lock

```diff
@@ -10,6 +10,6 @@ dependencies:
   version: 3.1.0
 - name: postgresql
   repository: oci://registry-1.docker.io/bitnamicharts
-  version: 12.5.6
-digest: sha256:e39405a7071d52fabbfb5c13dadabdd196cdfec40637bc26fd2444700c67d7e5
-generated: "2023-06-01T10:44:55.970183+02:00"
+  version: 12.6.6
+digest: sha256:2e35331d0c807c80675a67badca0eeaaef31538151957eabc4dfa904175049cf
+generated: "2023-07-18T13:01:41.970559642Z"
```

Comment:
please rebase or amend&pushforce or add an empty commit - these are untested

we probably need to move to alfresco-build PAT